input/fsh/questionnaires/MeaslesHealthCheckQuestionnaire.fsh

```diff
@@ -0,0 +1,191 @@
+* item[=].item[=].extension[=].valueDate.extension.valueExpression.language = #text/fhirpath
+* item[=].item[=].extension[=].valueDate.extension.url = "http://hl7.org/fhir/StructureDefinition/cqf-expression"
+* item[=].item[=].extension[=].url = "http://hl7.org/fhir/StructureDefinition/maxValue"
+* item[=].item[+].type = #text
+* item[=].item[=].linkId = "p02-q01-1-3-4-Symptoms.RashWhereSpread"
+* item[=].item[=].prefix = "page 2 question 1.1.3"
+* item[=].item[=].text = "Where on your body did the rash start? If the rash has spread, provide details of where it has spread to? (required)"
+* item[=].item[=].enableWhen.question = "p02-q01-1-Symptoms.Which"
+* item[=].item[=].enableWhen.operator = #=
+* item[=].item[=].enableWhen.answerCoding.display = "Red blotchy rash"
+* item[=].item[=].enableBehavior = #all
+* item[=].item[=].required = true
+* item[=].item[=].repeats = false
+* item[=].item[+].type = #text
+* item[=].item[=].linkId = "p02-q01-1-5-Symptoms.Which.Other"
+* item[=].item[=].prefix = "page 2 question 1.1.4"
+* item[=].item[=].text = "Provide details of your other symptoms: (required)"
+* item[=].item[=].enableWhen.question = "p02-q01-1-Symptoms.Which"
+* item[=].item[=].enableWhen.operator = #=
+* item[=].item[=].enableWhen.answerCoding.display = "Other"
+* item[=].item[=].enableBehavior = #all
+* item[=].item[=].required = true
+* item[=].item[=].repeats = false
+* item[=].item[+].type = #boolean
+* item[=].item[=].linkId = "p02-q01-2-Symptoms.Worsened"
+* item[=].item[=].prefix = "page 2 question 1.2"
+* item[=].item[=].text = "Have any of your symptoms worsened since your last health check? (required)"
+* item[=].item[=].enableWhen.question = "p02-q01-Symptoms"
+* item[=].item[=].enableWhen.operator = #=
+* item[=].item[=].enableWhen.answerBoolean = true
+* item[=].item[=].enableBehavior = #all
+* item[=].item[=].required = true
+* item[=].item[=].repeats = false
+* item[=].item[+].type = #text
+* item[=].item[=].linkId = "p02-q01-2-1-Symptoms.Worsened.Details"
+* item[=].item[=].prefix = "page 2 question 1.2.1"
+* item[=].item[=].text = "What are the details of your worsening symptoms? (required)"
+* item[=].item[=].enableWhen.question = "p02-q01-2-Symptoms.Worsened"
+* item[=].item[=].enableWhen.operator = #=
+* item[=].item[=].enableWhen.answerBoolean = true
+* item[=].item[=].enableBehavior = #all
+* item[=].item[=].required = true
+* item[=].item[=].repeats = false
+* item[=].type = #group
+* item[=].linkId = "p02"
+* item[=].prefix = "page 2"
+* item[=].text = "Symptoms"
+* item[+].item.type = #choice
+* item[=].item.linkId = "p03-q01-Contact"
+* item[=].item.prefix = "page 3 question 1"
+* item[=].item.text = "Do you know if you have had any contact with someone with measles since your last health check? (required)"
+* item[=].item.required = true
+* item[=].item.repeats = false
+* item[=].item.answerOption[0].valueCoding.display = "Yes"
```

Comment:
There is a ValueSet already in the IG covering these 3 options.  You could use it like this
`* item[=].item.answerValueSet = Canonical(QualifiedYesNoAnswerValueSet)    `